api/features_api.py

```diff
@@ -29,9 +29,25 @@ class FeaturesAPI(basehandlers.APIHandler):
 
   def do_get(self):
     user = users.get_current_user()
-    feature_list = models.Feature.get_chronological(
-        version=2,
-        show_unlisted=permissions.can_edit_feature(user, None))
+    feature_list = None
+
+    # Query-string parameter 'milestone' is provided
+    if self.request.args.get('milestone') is not None:
+      try:
+        milestone = int(self.request.args.get('milestone'))
+        feature_list = models.Feature.get_in_milestone(
+          version=2,
```

Comment:
You could drop the version=2 argument and the parameter in the function because they are not used.  I think that was a left-over detail of some old upgrade to the chronological feature list feed.  We don't need that for new code, especially since we have a version number in the API URL path.

Reply:
If I do not specify `version = 2`, then the API response will be different than what it is for `features.json` and `features_v2.json`. (Both `features.json` and `features_v2.json` use `version = 2` and hence, their response is same.) 

**_How is the response different ?_**
If we do not specify `version = 2`, then 
- the keys in the JSON response will be different. 
- much more information will be present in the response.

Reply:
Should we make the `format_for_template()` method use `version = 2` by default ?

Reply:
Yeah, let's make version=2 the default, then it can be omitted from this code.

Reply:
Okay.
Done
See [this](https://github.com/GoogleChrome/chromium-dashboard/pull/1424/commits/8ce4987c4f193c24781d866a2a9449a44a40548a) commit